### ISSUE TITLE
allowed_option and options are not synced on startup

### ISSUE THREAD

**Issue by harish988** [douglascrockford/JSLint#257]
Various options are given at allowed_option and they can be changed. But allowed_option values are not used, the entire code refers to option only. So the sync is missing between option and allowed_option.

For example set the eval to be false (eval:false) at the allowed_option. It will not be reflected.

**Comment by douglascrockford** [comment]
I am sorry, I need more information. What exactly is the problem?

**Comment by kaizhu256** [comment]
@harish988 reading the code, `allowed_option` is a read-only config of "allowed" jslint-options.  the user is not supposed to edit it.

what you are allowed to do is pass an `option_object` to jslint, e.g.:

```js
let option_object;
option_object = {
    // allow "eval" function-call
    eval: true,
    // allow nodejs-specific global variables
    node: true
};
jslint(
    "console.log(\"hello world\");\n",
    // jslint will lookup option_object against internal allowed_option
    option_object
);
```
